tests/integration/golang/mlflow/artifact/get_artifact_s3_test.go

```diff
@@ -39,9 +39,16 @@ func (s *GetArtifactS3TestSuite) SetupTest() {
 	s3Client, err := helpers.NewS3Client(helpers.GetS3EndpointUri())
 	assert.Nil(s.T(), err)
 
+	//nolint:errcheck,gosec
+	helpers.CreateBuckets(s3Client)
```

Comment:
you're not checking the error (and the linter tells you about it before you silence it!)

Reply:
this error is not worth checking -- if it can't create because bucket already exists the test will pass, if it fails for some other reason the PutObject for the bucket will fail the test later

Reply:
What if it can't create the bucket, full stop? Then we'll need to hunt down the actual error. It's literally just adding an assert instead of a linter comment, what is there to win by not checking the error?

Reply:
it's a conditional assert -- sometimes this is okay (already exists)

Reply:
well it may fail at first until the teardown succeeds, but I think simple assert will work after that.